### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by biswasingh** [HarryVolek/PyTorch_Speaker_Verification#45]
Hi,

I see the model is overfitting on the mix of data I am using. Can you please suggest some regularisation technique which might help here? Shall I add more data?

Sun Jul  7 10:31:55 2019        Epoch:1700[30/79],Iteration:134251      Loss:0.0008     TLoss:0.9285

Sun Jul  7 10:31:58 2019        Epoch:1700[60/79],Iteration:134281      Loss:0.0011     TLoss:0.6694

Sun Jul  7 10:32:05 2019        Epoch:1701[30/79],Iteration:134330      Loss:0.0631     TLoss:0.4391

**Comment by nidhal1231** [comment]
@biswasingh  could you please explain me how did you know your model is overfitting . is it in relation with Tloss. I didn't understand the signification of Tloss in the model. Thank you.
